msteams-platform/tabs/how-to/developer-tools.md

```diff
@@ -28,18 +28,19 @@ While the web version and the desktop version of Teams are almost the same, ther
 
       :::image type="content" source="../../assets/images/dev-preview/devtools-right-click.png" alt-text="Screenshot shows the option to open DevTools from Windows desktop.":::
 
-    * On MacOS, select the Microsoft Teams icon in the Dock.
+    * On macOS, select the Microsoft Teams icon in the Dock.
 
-      :::image type="content" source="../../assets/images/dev-preview/mac-os-developer-tools.PNG" alt-text="Screenshot shows the option to open DevTools from MacOS dock.":::
+      :::image type="content" source="../../assets/images/dev-preview/mac-os-developer-tools.PNG" alt-text="Screenshot shows the option to open DevTools from macOS dock.":::
 
 The following example shows DevTools open and inspecting a tab configuration dialog:
 
-   :::image type="content" source="../../assets/images/dev-preview/tab-and-devtools.png" alt-text="Screenshot shows the Tab and DevTools."  lightbox="../../assets/images/dev-preview/tab-and-devtools.png":::
+   :::image type="content" source="../../assets/images/dev-preview/tab-and-devtools.png" alt-text="Screenshot shows the Tab and DevTools." lightbox="../../assets/images/dev-preview/tab-and-devtools.png":::
 
 ## Access DevTools from an Android device
 
 You can also enable the DevTools from the Teams Android client. To enable DevTools, you must:
 
+1. **Developer tools** for Android is available only on the beta version of Teams app. To join the beta version of Teams app, follow the instructions listed in [Get beta versions of apps.](https://support.google.com/googleplay/answer/7003180?hl=en#:~:text=Get%20beta%20versions%20of%20apps)
```

Comment:
Can we change this to DevTools instead of Developer tools

Reply:
Updated